types/ws/index.d.ts

```diff
@@ -179,7 +171,14 @@ declare class WebSocket extends EventEmitter {
     removeListener(event: string | symbol, listener: (...args: any[]) => void): this;
 }
 
+declare class WebSocketAlias extends WebSocket {}
```

Comment:
You can do this using TS namespace imports:
```suggestion
import WebSocketAlias = WebSocket;
```
And below,
```ts
export WebSocket = WebSocketAlias;
```

Reply:
Or

```suggestion
declare const WebSocketAlias: typeof WebSocket;
```